enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -208,18 +208,84 @@ if (hasPremiumFeature("whitelabel")) {
             key: "login-page-illustration",
             tab: "conceal-metabase",
             display_name: (
-              <LoginPageIllustrationDescription errorMessageContainerId="login-page-illustration-error-container" />
+              <IllustrationDescription
+                errorMessageContainerId="login-page-illustration-error-container"
+                type="login-page"
+              />
             ),
             description: null,
             type: "string",
             widget: IllustrationWidget,
             props: {
               defaultIllustrationLabel: t`Lighthouse`,
+              defaultPreviewType: "lighthouse",
               customIllustrationSetting: "login-page-illustration-custom",
               errorMessageContainerId:
                 "login-page-illustration-error-container",
             },
           },
+          {
+            key: "landing-page-illustration",
+            tab: "conceal-metabase",
+            display_name: (
+              <IllustrationDescription
+                errorMessageContainerId="landing-page-illustration-error-container"
+                type="landing-page"
+              />
+            ),
+            description: null,
+            type: "string",
+            widget: IllustrationWidget,
+            props: {
+              defaultIllustrationLabel: t`Lighthouse`,
+              defaultPreviewType: "lighthouse",
+              customIllustrationSetting: "landing-page-illustration-custom",
+              errorMessageContainerId:
+                "landing-page-illustration-error-container",
+            },
+          },
+          {
+            key: "no-question-results-illustration",
+            tab: "conceal-metabase",
+            display_name: (
+              <IllustrationDescription
+                errorMessageContainerId="no-question-results-illustration-error-container"
+                type="no-question-results"
+              />
+            ),
+            description: null,
+            type: "string",
+            widget: IllustrationWidget,
+            props: {
+              defaultIllustrationLabel: t`Sailboat`,
+              defaultPreviewType: "sailboat",
+              customIllustrationSetting:
+                "no-question-results-illustration-custom",
+              errorMessageContainerId:
+                "no-question-results-illustration-error-container",
```

Comment:
The feedback for props is welcome. Though, they'll keep changing in the upcoming PRs.